nodemanagement/node_management_worker.go

```diff
@@ -416,7 +416,8 @@ func (n *NodeManagementWorker) ProcessAllNMPS(baseWorkingFile string, getAllNMPS
 }
 
 func (n *NodeManagementWorker) NewEvent(incoming events.Message) {
-	glog.Infof(nmwlog(fmt.Sprintf("Handling event: %v", incoming)))
+	glog.Infof(nmwlog(fmt.Sprintf("Handling event type: %v", incoming.Event())))
+	glog.V(5).Infof(nmwlog(fmt.Sprintf("Handling event: %v", incoming)))
```

Comment:
Please change to

if glog.V(5) {glog.Infof(nmwlog(fmt.Sprintf("Handling event: %v", incoming)))}